app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,40 @@
+$color: #EC4339;
+
+.title-box {
+   background-color: $color;
+   font-family: 'Alegreya Sans SC', sans-serif;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered background-color, color, font-family, font-size, font-size, font-weight, letter-spacing, padding